runtime/compiler/aarch64/runtime/PicBuilder.spp

```diff
@@ -1104,3 +1105,12 @@ FUNC_LABEL(_nativeStaticHelper):
 	ldr	x30, [x30, #J9TR_SCSnippet_codeCacheReturnAddress]	// get code cache return address
 	and	x0, x0, #(~clinit_bit)				// clear the "<clinit>" bit
 	b	FUNC_LABEL(icallVMprJavaSendNativeStatic)	// jump to VM helper
+
+// x0 holds address of the compare instruction
+// trashes x1
+FUNC_LABEL(_patchGCRHelper):
+	ldr w1, [x0]			// load compare instruction
+	orr w1, w1, #2048		// set bit 11 (change immediate to 3)
+	str w1, [x0]			// write updated compare instruction
```

Comment:
Please use a tab character between the opcode and the operands.

Reply:
Changed those lines to use a tab character.